docs/jade-getting-started.md

```diff
@@ -52,6 +52,18 @@ relevant to your team.
 Connect your Docker Hub account to your Broad profile by contacting the DevOps
 team.
 
+### 3. Create Terra Accounts
+The Data Repo uses [Sam](https://github.com/broadinstitute/sam) for identity and access management. To register
+as a new user, create an account through Terra. Use a non-Broad email for the non-prod instances:
```

Comment:
> Use a non-Broad email for the non-prod instances

I think we should stress that this is an email account that is designed specifically for development purposes as opposed to a personal email account.